resources/scheduled_task.rb

```diff
@@ -53,13 +53,15 @@
                    "#{cmd_path} /c \'#{client_cmd}\'"
                  end
 
+  # According to https://docs.microsoft.com/en-us/windows/desktop/taskschd/schtasks,
+  # the :once, :onstart, :onlogon, and :onidle schedules don't accept schedule modifiers
   windows_task new_resource.task_name do
     run_level :highest
-    command full_command
+    command   full_command
```

Comment:
Can you just fully line these up. Rubocop looks like it's doing some strange things.

Reply:
Weird. It's lined up in my editor, and `git status` says it's up to date

Reply:
Oh, with the following lines. New commit coming